### ISSUE TITLE
White Issue and PR title headers

### ISSUE THREAD

**Issue by jodizzle** [StylishThemes/GitHub-Dark#787]
* **Browser**: Firefox
* **Operating System**: Linux
* **Screenshot**: 
![2018-12-06-150555_891x100_scrot](https://user-images.githubusercontent.com/672268/49609065-75055980-f968-11e8-84bf-388f166db6f6.png)

* **HTML of the section where the issue occurs**:
```html
<div class="gh-header-hide-if-sticky">
    <div class="gh-header-show ">
        <div class="gh-header-actions">

              <a href="/StylishThemes/GitHub-Dark/issues/new" class="btn btn-sm btn-primary float-right" data-hotkey="c" data-ga-click="Issues, create new issue, view:issue_show location:issue_header style:button logged_in:true">
                New issue
              </a>
        </div>

      <h1 class="gh-header-title">
        <span class="js-issue-title">
          GitHub Actions
        </span>
        <span class="gh-header-number">#782</span>
      </h1>
    </div>


    <div class="TableObject gh-header-meta">
      <div class="TableObject-item">
          <div class="State State--green">
    <svg class="octicon octicon-issue-opened" viewBox="0 0 14 16" version="1.1" width="14" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7 2.3c3.14 0 5.7 2.56 5.7 5.7s-2.56 5.7-5.7 5.7A5.71 5.71 0 0 1 1.3 8c0-3.14 2.56-5.7 5.7-5.7zM7 1C3.14 1 0 4.14 0 8s3.14 7 7 7 7-3.14 7-7-3.14-7-7-7zm1 3H6v5h2V4zm0 6H6v2h2v-2z"></path></svg>
    Open
  </div>

      </div>
      <div class="TableObject-item TableObject-item--primary">
          <a class="author" data-hovercard-type="user" data-hovercard-url="/hovercards?user_id=831974" data-octo-click="hovercard-link-click" data-octo-dimensions="link_type:self" href="/xt0rted" aria-describedby="hovercard-aria-description">xt0rted</a>  opened this <span class="noun">Issue</span>
          <relative-time datetime="2018-12-04T21:04:47Z" title="Dec 4, 2018, 4:04 PM EST">2 days ago</relative-time>
          · 11 comments
      </div>
    </div>
  </div>
```

**Comment by mathieumg** [comment]
Same with Chrome 70 on Mac.

**Comment by skid9000** [comment]
I can confirm the issue, tested on latest version of Firefox and Chromium on Windows and Linux.

**Comment by Ash258** [comment]
Update to latest version.

This should be fixed in 1.20.86 version. (acc76270b620027e7e4adeb98421305b00b1278b)

**Comment by jodizzle** [comment]
Fixed for me as well.  I had previously installed from userstyles.org, which apparently doesn't auto-update.  I'm on the usercss now.
